### PR TITLE
Bump Netty to 4.1.86.Final to fix CVE-2022-41881

### PR DESCRIPTION
**Pull request by MarcusWalzCollibra**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
     <!--Version of grpc to be shaded -->
     <shaded.grpc.version>1.48.1</shaded.grpc.version>
     <!--Version of Netty to be shaded -->
-    <shaded.netty.version>4.1.77.Final</shaded.netty.version>
+    <shaded.netty.version>4.1.86.Final</shaded.netty.version>
```
Comment (szetszwo) [inline pom.xml:75]: @MarcusWalzCollibra , thanks for working on this!   Let's also bump grpc version accordingly, see
https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Comment (MarcusWalzCollibra) [reply pom.xml:75]: @szetszwo just bumped grcp! Thank you

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Comment by szetszwo** [comment]
@MarcusWalzCollibra , could you comment on https://issues.apache.org/jira/browse/RATIS-1760 so I can assign it to you?

BTW, please file a JIRA when submitting a pull request.

**Comment by szetszwo** [comment]
@MarcusWalzCollibra , could you post a comment on https://issues.apache.org/jira/browse/RATIS-1760 so that we can assign it to with your JIRA account?

**Pull request merged**